### PR TITLE
Add Dask 2.0 draft

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

**Review by TomAugspurger — APPROVED** [review]

File: _posts/2019-06-22-dask-2.0.md [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+
+Python 2 reaches end of life in 2020, just six months away.  Most major PyData
+projects are dropping Python 2 support around now.  See the [Python 3
+Statement](https://python3statement.org/) for more details about some of your
+favorite projects.
+
+Python 2 users can continue to use older versions of Dask, which are in
+widespread use today.  Institutions looking for long term support of Dask in
+Python 2 may wish to reach out to some for-profit consulting companies, like
+[QuanSight](https://www.quansight.com/)
```
Comment (jrbourbeau) [inline _posts/2019-06-22-dask-2.0.md]:
```suggestion
[Quansight](https://www.quansight.com/).
```

File: _posts/2019-06-22-dask-2.0.md [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+### Black everywhere
+
+We now use the [Black](https://black.readthedocs.io/) code formatter throughout
+most Dask repositories.  These repositories include pre-commit hooks, which we
+recommend when developing on the project.
+
+```
+cd /path/to/dask
+git checkout master
+git pull origin master
```
Comment (jrbourbeau) [inline _posts/2019-06-22-dask-2.0.md]: Nitpick: should this be `upstream`?

```suggestion
git pull upstream master
```

File: _posts/2019-06-22-dask-2.0.md [hunk 3]
```diff
@@ -0,0 +1,480 @@
...
+
+-  The distributed.bokeh module has moved to distributed.dashboard
+-  Various `ncores` keywords have been moved to `nthreads`
+-  Client.map/gather/scatter no longer accept iterators and Python queue
+   objects.  Users can handle this themselves with `submit`/`as_completed` or
+   can use the [Streamz](https://github.com/python-streamz/streamz) library.
+-  The worker `/main` route has moved to `/status`
+-  Cluster.workers is not a dictionary mapping worker name to worker, rather
+   than a list as it was before
+-  TODO
```
Comment (jrbourbeau) [inline _posts/2019-06-22-dask-2.0.md]: Just a reminder that this TODO is still here

File: _posts/2019-06-22-dask-2.0.md [hunk 4]
```diff
@@ -0,0 +1,480 @@
...
+
+## Install
+
+As always, you can conda install Dask:
+
+    conda install dask
+
+or pip install from PyPI:
+
+    pip install dask[complete] --upgrade
```
Comment (jrbourbeau) [inline _posts/2019-06-22-dask-2.0.md]:
```suggestion
    pip install "dask[complete]" --upgrade
```

File: _posts/2019-06-22-dask-2.0.md [hunk 5]
```diff
@@ -0,0 +1,480 @@
...
+
+We now include a list with a brief description of most of the breaking changes:
+
+-  The distributed.bokeh module has moved to distributed.dashboard
+-  Various `ncores` keywords have been moved to `nthreads`
+-  Client.map/gather/scatter no longer accept iterators and Python queue
+   objects.  Users can handle this themselves with `submit`/`as_completed` or
+   can use the [Streamz](https://github.com/python-streamz/streamz) library.
+-  The worker `/main` route has moved to `/status`
+-  Cluster.workers is not a dictionary mapping worker name to worker, rather
```
Comment (jrbourbeau) [inline _posts/2019-06-22-dask-2.0.md]:
```suggestion
-  Cluster.workers is now a dictionary mapping worker name to worker, rather
```

**Review by jrbourbeau — COMMENTED** [review]
Thank you for writing this blog post @mrocklin, it's a really nice summary. I've attached a few small comments. 

Looking forward to the 2.0 releases : )

**Review by jrbourbeau — APPROVED** [review]

**Comment by jrbourbeau** [comment]
Is this good to merge?

**Pull request merged**